itest/lib/app.js

```diff
@@ -52,6 +53,9 @@ export const newAppInstance = (name: string, idx: number): Application => {
   if (isCI() && process.platform === "darwin") {
     appArgs = {...appArgs, path: macInstallPath}
     LOG.debug("Chose installed MacOS app location", macInstallPath)
+  } else if (isCI() && process.platform === "linux") {
+    appArgs = {...appArgs, path: debianInstallPath, args: [repoDir()]}
+    LOG.debug("Chose working copy app location", electronPath)
```

Comment:
```suggestion
  } else if (isCI() && process.platform === "linux") {
    appArgs = {...appArgs, path: linuxInstallPath}
    LOG.debug("Chose installed Linux app location", linuxInstallPath)
```

Reply:
Explanation for the above: We want to run just `/usr/bin/brim`, not `/usr/bin/brim /home/runner/brim`. The latter might point to the working copy, not the installed app. Note how in the darwin branch above, `args` is not included either.